go/tasks/pluginmachinery/core/exec_metadata.go

```diff
@@ -32,4 +32,5 @@ type TaskExecutionMetadata interface {
 	GetLabels() map[string]string
 	GetAnnotations() map[string]string
 	GetK8sServiceAccount() string
+	GetInterruptible() bool
```

Comment:
I like "IsInterruptible" ("Is...") for boolean properties.

Reply:
Updated.

Reply:
Thank you.